zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4441,6 +4463,63 @@ components:
         email:
           description: The email of the user.
           type: "string"
+    CamundaUser:
+      type: "object"
+      properties:
+          userId:
+            description: The ID of the user.
+            type: "string"
+          userKey:
+            description: The system generated key of the User
+            type: "integer"
+            format: "int64"
+          displayName:
+            description: The display name of the User
+            type: "string"
+          authorizedApplications:
+            description: The applications the User is authorized to use
+            type: array
+            items:
+              type: "string"
+          tenants:
+            description: The tenants the User is a member of
+            type: array
+            items:
+              type: "object"
+              properties:
+                tenantId:
+                  type: "string"
+                name:
+                  type: "string"
+          groups:
+            description: The groups assigned to the user
+            type: array
+            items:
+              type: "string"
```

Comment:
Are these keys or names? I believe we don't have an immutable identifier like the tenant id for roles and groups.

Reply:
These are names :)